motivaa-backend/motivaa-app/src/test/java/backendtesting/RequestMaker.java

```diff
@@ -2,11 +2,27 @@
 
 import io.restassured.RestAssured;
 
+import java.util.List;
 import java.util.Map;
 
 public class RequestMaker {
 
-    public static String initiateGetRequest(String host,
+    public static ProcessObject createProcessObject(
+            String environmentHost,
+            String processStartEndpoint,
+            String jSessionId) {
+        String processStart_responseBody = RequestMaker.initiateGetRequest(
```

Comment:
this processStart_responseBody naming is not consistent with the others, shouldn't it be processStartResponseBody?